committee_admissions/oauth.py

```diff
@@ -20,15 +20,15 @@ class LegoOAuth2(BaseOAuth2):
     ]
 
     LEGO_GROUP_NAMES = [
```

Comment:
Are we not missing "Koret" from this list?

Reply:
Ooo that's true, if that is the case and they don't go under Band then they are missing from the groups in LEGO as well..

Reply:
I just checked https://abakus.no/pages/grupper/104-revyen